xbmc/cores/IPlayer.h

```diff
@@ -28,6 +28,7 @@
 #include "VideoSettings.h"
 #include "guilib/Geometry.h"
 #include "guilib/Resolution.h"
+#include "VideoPlayer/DVDDemuxers/DVDDemux.h"
```

Comment:
any definition from nested folders is not supposed to be exposed to the world. there is a new folder "Interface". please move/create required definitions to this folder.

Reply:
I didn't like it and wanted to ask you anyway, thx for the hint.